### DISCUSSION TITLE
!Clear:AutoSwitch line causing Quiffen to ignore the first account

### DISCUSSION THREAD

**Discussion by dshovel** [isaacharrisholt/quiffen#89] [Q&A] [answered]
I'm not a developer nor am I an experience GH user so please excuse me if I am asking a question that has an obvious answer or if this is asked in the wrong forum.

I noticed that Quiffen creates a default account instead of using the actual account details when I parse my QIF file.  I removed the line containing !Clear:AutoSwitch just above the first line containing !Account and this enabled the behavior I was looking for when I ran the parse function.

Is there a way to have the Parse function ignore this line instead of having to manually delete it?

Thanks!

**Comment by isaacharrisholt** [comment]
Hey @dshovel, thanks for reporting this. `!Clear:` does not seem to be part of the QIF standard, and I'm not really sure what behaviour `!Clear:AutoSwitch` should have. Where did you export this file from?

**Reply by dshovel** [reply]
I am using Quicken Classic desktop version.  
Version R56.9
Build 27.1.56.9

I have created a brand new Quicken file with one bank account for use while learning how to use Quiffen.  I created my QIF file using the export feature in Quicken.

It seems like !Clear is used by GNUCash.  I found this reference on [StackOverflow](https://stackoverflow.com/questions/35385935/how-can-i-map-my-transactions-to-accounts-in-a-qif-file).

**Reply by isaacharrisholt** [reply]
Interesting. It should be easy enough to ignore lines with `!Clear`, since they should have no impact on Quiffen's behaviour. Let me try something.

**Comment by isaacharrisholt** [comment, marked as answer]
This should be fixed as of Quiffen v2.0.11. Thanks for bringing it to my attention!
